recipes/libaec/all/patches/1.0.6-0002-fix-cmake-build-with-ninja.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt	2021-09-17 19:28:01.000000000 +0800
++++ b/CMakeLists.txt	2022-11-04 07:43:58.413439439 +0800
+@@ -12,6 +12,8 @@
+ include(TestBigEndian)
+ test_big_endian(WORDS_BIGENDIAN)
+ 
++set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
++
```

Comment:
Is it also needed in 1.0.4?

Reply:
Good question, no it isn't necessary.
The CMake files changed between 1.0.4 and 1.0.6, and the older one didn't have that problem.

Thanks for asking though, I did find a different problem related to ordering of lib files.
The linker needed libsz.a to come before libaec.a